docs/sources/tempo/api_docs/metrics-summary.md

```diff
@@ -10,15 +10,16 @@ weight: 600
 # Metrics summary API
 
 {{< admonition type="warning" >}}
-The Metrics summary API is an [experimental feature](/docs/release-life-cycle) that is disabled by default. To enable it, adjust your configuration as suggested below.
+Metrics summary API is deprecated in Grafana Cloud and Grafana 11.3 and later.
```

Comment:
since these are OSS tempo docs should we mention a tempo version (2.7?) instead of a grafana version and cloud?

Reply:
Good question. I removed the mention of Aggregate by because we don't discuss the UI here. What if we changed this to something like: 
```suggestion
Metrics summary API is deprecated in Tempo 2.7. Features powered by metrics summary API, like the Aggregate by table, are also deprecated in Grafana Cloud and Grafana 11.3 and later.
```

Reply:
i like that 👍